cgmes/cgmes-conversion/src/test/resources/cim14/nordic32.xiidm

```diff
@@ -90,7 +90,7 @@
                 <iidm:bus id="_N1022____TN" v="110.60600280761719" angle="-17.621028900146484"/>
             </iidm:busBreakerTopology>
             <iidm:load id="_N1022____EC" name="N1022   " loadType="UNDEFINED" p0="280.0" q0="95.0" bus="_N1022____TN" connectableBus="_N1022____TN" p="280.0" q="95.0"/>
-            <iidm:shunt id="_N1022____SC" name="N1022   " bPerSection="0.002958579920232296" maximumSectionCount="1" currentSectionCount="1" voltageRegulatorOn="false" bus="_N1022____TN" connectableBus="_N1022____TN"/>
+            <iidm:linearShunt id="_N1022____SC" name="N1022   " bPerSection="0.002958579920232296" gPerSection="0.0" maximumSectionCount="1" currentSectionCount="1" voltageRegulatorOn="false" bus="_N1022____TN" connectableBus="_N1022____TN"/>
```

Comment:
I would not change the top level equipment name. This is really confusing.
Why not:
```xml
<iidm:shunt ...>
    <iidm:shuntLinearModel ... />
```

Reply:
I started like this but it made the file bigger and less understandable? (particularly for linear shunts, it really make the XML object bigger and I didn't really think it was justified)